server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -51,9 +51,11 @@ export interface ICollection<T> {
      *
      * @param query - data we want to find
      * @param sort - object with property we use to sort on, whose value is 0 for descending order and 1 for ascending
+     * @param limit - optional, if set, query result will set less equal to this number. Our mongo layer internally used 2000.
```

Comment:
nit: "optional - if set, limits the number of documents/records the cursor will return"

Borrowed and updated the description from here: https://www.mongodb.com/docs/drivers/node/v4.6/fundamentals/crud/read-operations/limit/?_ga=2.68335960.811589495.1653330422-1860421701.1644639209

Reply:
Addressed. Kept the `2000` part.